volumes.xconf

```diff
@@ -15,8 +15,8 @@
             <text qname="tei:div">
                 <field name="within" expression="if (@type = ('section', 'document')) then 'documents' else ()"/>
                 <field name="content" expression="if (@type = ('section', 'document')) then . else ()"/>
-                <field name="date-min" expression="if (@type = ('section', 'document')) then @frus:doc-dateTime-min else ()"/>
-                <field name="date-max" expression="if (@type = ('section', 'document')) then @frus:doc-dateTime-max else ()"/>
+                <field name="date-min" expression="if (@type = ('section', 'document')) then translate(@frus:doc-dateTime-min, ':-', '') else ()"/>
```

Comment:
Have you considered switching the date fields to use the `keyword` analyzer? This would avoid the need for `translate()`. Also, since the field already has an implicit `@type="string"`, Lucene range queries don't have a particular need for the values to be numeric. In hsg-facets I used: `<analyzer class="org.apache.lucene.analysis.core.KeywordAnalyzer" id="keyword"/>` and `<field name="date-min" expression="./@frus:doc-dateTime-min" type="xs:string" analyzer="keyword"/>`.